driver/src/main/scala/com/datastax/spark/connector/mapper/MappedToGettableDataConverter.scala

```diff
@@ -208,7 +208,7 @@ object MappedToGettableDataConverter extends StrictLogging {
           for (i <- columnValues.indices)
             columnValues(i) = converters(i).convert(columnValues(i))
           struct.newInstance(columnValues: _*)
-        case None =>
+        case _ =>
```

Comment:
Maybe i'm just paranoid, but should we do None | null? I may just be paranoid (this wouldn't be the first time...)

Reply:
I would think that the _ wildcard symbol catches both None and null as it matches anything:

> case _ =>         // Wild card pattern -- matches anything

Source: https://docs.scala-lang.org/tutorials/FAQ/finding-symbols.html